tsconfig.json

```diff
@@ -12,9 +13,10 @@
         "baseUrl": "./src",
         "preserveSymlinks": true,
         "paths": {
-            "*": ["../node_modules/*"]
-        }
+          "*": ["../node_modules/*"]
+        },
+        "types": ["node"],
+        "lib": ["es2015", "dom"]
     },
     "include": ["src/**/*.tsx", "src/**/*.ts", "custom.d.ts"],
```

Comment:
you have issue with types because you haven't added Kaltura types. Example:
https://github.com/kaltura/playkit-js-moderation/blob/master/tsconfig.json#L21